bookwyrm/templates/notifications/items/fav.html

```diff
@@ -16,23 +16,74 @@
     {% with related_status.local_path as related_path %}
 
     {% if related_status.status_type == 'Review' %}
-    {% blocktrans trimmed %}
+        {% if other_user_count == 0 %}
```

Comment:
It would be more DRY but unfortunately it would be impossible to translate well, which is why these very similar strings are just repeated over and over